src/Components/Artwork/Details/BidTimerLine.tsx

```diff
@@ -13,22 +15,30 @@ interface BidTimerLineProps {
 export const BidTimerLine: React.FC<BidTimerLineProps> = ({ artwork }) => {
   const data = useFragment(bidTimerLineFragment, artwork)
 
-  const { collectorSignals } = data
+  const { collectorSignals, saleArtwork } = data
   const { lotClosesAt, registrationEndsAt, onlineBiddingExtended } =
     collectorSignals?.auction ?? {}
-  const { time } = useTimer(lotClosesAt ?? "")
+
+  const [updatedLotClosesAt, setUpdatedLotClosesAt] = useState(lotClosesAt)
+
+  useAuctionWebsocket({
+    lotID: saleArtwork?.lotID!,
+    onChange: ({ extended_bidding_end_at }) => {
+      setUpdatedLotClosesAt(extended_bidding_end_at)
+    },
+  })
+
+  const { time, hasEnded: hasBiddingEnded } = useTimer(updatedLotClosesAt!)
```

Comment:
Two updates here - use `hasEnded` to avoid the `new Date()` check below - this hook will already use the system time (not local time). And, pass in `updatedLotClosesAt`, which comes from local state and can be updated by websocket.